langchain/src/tools/tests/discord.int.test.ts

```diff
@@ -0,0 +1,59 @@
+import { test } from "@jest/globals";
+import {
+  DiscordGetMessagesTool,
+  DiscordChannelSearchTool,
+  DiscordSendMessagesTool,
+  DiscordGetGuildsTool,
+  DiscordGetTextChannelsTool,
+} from "../discord.js";
+
+test.skip("DiscordGetMessagesTool", async () => {
+  const tool = new DiscordGetMessagesTool();
+
+  try {
+    const result = await tool.invoke("1153400523718938780");
+    console.log(result);
+  } catch (error) {
+    console.error(error);
+  }
+});
```

Comment:
Should have at the bare minimum `expect(result).toBeThruthy()` but ideally is more involved than that. Please update for the other tests.